harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -99,6 +139,406 @@ def test_minimal_vm(api_client, image_id, unique_vm_name, wait_timeout):
         )
 
 
+@pytest.mark.p0
+@pytest.mark.virtualmachines
```

Comment:
this should be marked as negative test

Reply:
I did forget that marker, great catch :smile: - updated :+1: